src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -403,9 +403,9 @@ private void updateSupplementaryEvidence(
         );
 
         log.info(
-            "Successfully attach Exception Record with ID {} to case with id {}",
-            theCase.getId(),
-            exceptionRecordDetails.getId()
+            "Attached Exception Record to a case in CCD. ER ID: {}. Case ID: {}",
+            exceptionRecordDetails.getId(),
+            theCase.getId()
```

Comment:
Wrong argument order was the main reason for this PR

Reply:
I hope some of the reports that I had to do were not based on these log entries 😬